src/Microsoft.Restier.EntityFramework/Model/ModelMapper.cs

```diff
@@ -60,10 +61,10 @@ public ModelMapper(Type dbContextType)
             if (property != null)
             {
                 var type = property.PropertyType;
-                if (type.IsGenericType &&
-                    type.GetGenericTypeDefinition() == typeof(DbSet<>))
+                var parent = GetGenericParent(type);
```

Comment:
I pushed the changes on `FindGenericType` method. However, `TypeExtensions` class appears to be marked internal, as it is defined in `System` namesapce. So, i couldn't use it from the `ModelMapper` class. It seems to me, the best solution is moving `TypeExtensions` class out from `System` namespace and make it public. However, as it will effect all places where the method is used, i didn't take any action.